extensions/data-plane-transfer/data-plane-transfer-client/README.md

```diff
@@ -0,0 +1,37 @@
+# Data Plane Transfer Client extension
+
+## Overview
+
+The `data-plane-transfer-client` extension provides resources enabling to delegate a data transfer to the Data Plane
+through its Control API. This extension provides a `DataFlowController` implementation
+called `DataPlaneTransferFlowController` which is triggered for any destination type in `DataRequest`,
+except `HttpProxy` which is reserved for synchronous data transfers (see `data-plane-transfer-sync` for more details).
+This flow controller first resolves the data address based on the asset id coming from the `DataRequest` and then map
+a `DataFlowRequest` out of it. This `DataFlowRequest` is finally forwarded to the Data Plane, which takes care of the
+actual data transfer.
+
+There are two possible configurations:
+
+- Data Plane is embedded in the Control Plane runtime: this is mostly a configuration used for testing, demo...
+- Data Plane "lives" in a separated runtime (generally in the same cluster): this is the typical production
+  configuration.
+
+## Data Plane is embedded in Control Plane runtime
+
+The cornerstone of the Data Plane is the `DataPlaneManager`. Thus, we rely on the presence/absence of this service in
```

Comment:
Could you specify what needs to be done to enable embedded VS remote transfer clients from an EDC client perspective? Which extensions need to be in place or absent in each case?